examples/tv-app/android/App/common-api/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
what gets built here? Can we add a comment on why this gitignore is needed?

Same question for the other gitignore files

Reply:
Removed the unused gitignore. Others are required because when developing in android studio the modules are built separately and build files are created in the build folder in the module.